provisioning/stream/kubernetes/templates/config/config-map.yaml

```diff
@@ -0,0 +1,195 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: stream-config
+  namespace: $NAMESPACE
+  labels:
+    app: stream
+data:
+  keboolaStack: "$KEBOOLA_STACK"
+  keboolaRevision: "$RELEASE_ID"
+  config.yaml: |
+    ####################################################################################################################
+    # Notes:
+    #  - Commented keys are set via ENV.
+    #  - Each key can be set via ENV, for example "storage.volumesPath" is "STREAM_STORAGE_VOLUMES_PATH" ENV.
+    #  - Default configuration can be generated via "<stream binary> --dump-config=yaml > config.yaml".
+    ####################################################################################################################
+    # Enable logging at DEBUG level.
+    debugLog: false
```

Comment:
There is a lot of configuration options, so we are using configuration file.
The keys can be also defined via ENVs, and ENVs have priority.
Final configuration is logged to the DD on the each container start, so it can be easily checked,